sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -5,7 +5,7 @@
 import logging
 import threading
 
-from typing import Any, Union, TYPE_CHECKING, Iterable, Dict
+from typing import Any, Union, TYPE_CHECKING, Iterable, Dict  # pylint: disable=unused-import
```

Comment:
I just merged a PR that has the pylint fix on this file. So you may revert the changes on this file or accept my changes and merge.